database/sql/V072__auction_solution_orders.sql

```diff
@@ -0,0 +1,53 @@
+-- Rename current `auctions` table to `auction` since it always contains only the latest auction
+ALTER TABLE auctions RENAME TO auction;
+
+CREATE TABLE auctions (
+   auction_id bigint PRIMARY KEY,
+   -- The block number at which the auction was created
+   block bigint NOT NULL,
+   -- The block number until which all winning solutions from a competition should be settled on-chain
+   deadline bigint NOT NULL
+);
+
+-- Table to store all proposed solutions for an auction, received from solvers during competition time.
+-- A single auction can have multiple solutions, and each solution can contain multiple order executions.
+-- This design allows for multiple solutions from a single solver
+CREATE TABLE proposed_solutions (
+   auction_id bigint NOT NULL,
+   -- solver submission address
+   solver bytea NOT NULL,
+   -- Has to be unique accross auctions (hash of the (auction_id + solver + solutionId received from solver)?)
+   solution_id numeric NOT NULL,
+   -- Whether the solution is one of the winning solutions of the auction
+   is_winner boolean NOT NULL,
```

Comment:
Isn't this field something that can be derived in the code from the data? I'm not sure this should be persisted.

Reply:
Already discussed here:

https://github.com/cowprotocol/services/pull/2980#discussion_r1762985497